### PR TITLE
Add production flag to preversion

### PR DESCRIPTION
**Pull request by paulathevalley**
This PR adds the production flag to the `npm:preversion` task. This will use the correct template, fix the `<base href="">`, and use the proper `<title>` tags per page. 

/cc @ryan-roemer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "lint-config": "eslint ./*.js",
     "lint-react": "eslint --ext .js,.jsx src",
     "lint-test": "eslint --ext .js,.jsx test",
-    "npm:preversion": "builder run lint && builder run build-static",
+    "npm:preversion": "builder run lint && NODE_ENV=production builder run build-static",
```
Comment (ryan-roemer) [inline package.json]: Prefer portable variables, so:

``` js
"npm:preversion": "builder run lint && builder --env='{"NODE_ENV":"production"}' run build-static",
```

**Review by ryan-roemer — CHANGES_REQUESTED** [review]

**Review by ryan-roemer — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "lint-config": "eslint ./*.js",
     "lint-react": "eslint --ext .js,.jsx src",
     "lint-test": "eslint --ext .js,.jsx test",
-    "npm:preversion": "builder run lint && builder run build-static",
+    "npm:preversion": "builder run lint && --env='{"NODE_ENV":"production"}' builder run build-static",
```
Comment (ryan-roemer) [inline package.json]: Sorry -- you need to escape the `"` characters.

Also, can you check in a repo with running  `npm run npm:preversion` ?

Comment (paulathevalley) [reply package.json]: Oops, fixed.

What do you mean by your last question? The task will create a `build` that doesn’t get checked into github, right? So, what am I checking in?

Comment (ryan-roemer) [reply package.json]: @paulathevalley -- Actually hook in the changed archetype and in a project run `npm run npm:preversion` and verify it creates the `build/` directory.

**Comment by paulathevalley** [comment]
Confirmed, it works! (pushing now)

**Comment by ryan-roemer** [comment]
@paulathevalley -- Cool! Just make sure CI passes and you're good to merge, publish and roll out to all repos!

**Pull request merged**